public/video-ui/src/components/ReactApp.js

```diff
@@ -14,6 +14,7 @@ class ReactApp extends React.Component {
   componentWillMount() {
     if (this.props.params.id) {
       this.props.appActions.getVideo(this.props.params.id);
+      this.props.appActions.getPublishedVideo(this.props.params.id);
```

Comment:
We don't, I tried combining them into a single one originally, but in the end I thought that this would be clearer.